tests/snapshots/test_snapshot.py

```diff
@@ -3,7 +3,6 @@
 import tempfile
 from copy import deepcopy
 import pandas as pd
-from pandas.util.testing import assert_frame_equal
```

Comment:
From what I could tell, this was not being used and it apparently is getting deprecated as well: https://github.com/pydata/pandas-datareader/issues/775

Reply:
yup